digdag-standards/src/main/java/io/digdag/standards/operator/aws/S3WaitOperatorFactory.java

```diff
@@ -100,6 +103,9 @@ public TaskResult run()
             Optional<String> key = params.getOptional("key", String.class);
             Optional<String> versionId = params.getOptional("version_id", String.class);
             Optional<Boolean> pathStyleAccess = params.getOptional("path_style_access", Boolean.class);
+            Optional<Duration> timeout = params.getOptional("timeout", String.class)
+                                            .transform( (t) -> Durations.parseDuration(t));
```

Comment:
parseDuration throws a RuntimeException which is unexpected.
You can use DurationParam instead which implements error handling already: https://github.com/treasure-data/digdag/blob/6547542a274ac1a30c88c82e509ac0742d6d872e/digdag-plugin-utils/src/main/java/io/digdag/util/DurationParam.java
Usage of it: https://github.com/treasure-data/digdag/search?q=DurationParam&unscoped_q=DurationParam

Reply:
Thanks! Will fix it.